src/test/java/seedu/address/logic/parser/FindContactCommandParserTest.java

```diff
@@ -26,6 +26,17 @@ public void parse_invalidArgs_throwsParseException() {
         // multiple search prefixes (e.g. both n/ and m/)
         assertParseFailure(parser, " n/Alice m/CS1101S",
                 String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindContactCommand.MESSAGE_USAGE));
+
+        // invalid prefixes
+        assertParseFailure(parser, "s/ foo",
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindContactCommand.MESSAGE_USAGE));
+
+        assertParseFailure(parser, "s/ foo g/ bar",
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindContactCommand.MESSAGE_USAGE));
```

Comment:
In general, invalid prefixes are specified to be ignored in the UG.

Also, is there a reason this is still using the space-after-prefix syntax?

Reply:
No reason, but I've changed to no space for consistency